eng/versioning/external_dependencies.txt

```diff
@@ -315,14 +315,14 @@ storage_com.microsoft.azure:azure-storage;8.6.6
 
 # sdk\spring\pom.xml modules
 springboot3_com.diffplug.spotless:spotless-maven-plugin;2.30.0
-springboot3_com.fasterxml.jackson.core:jackson-annotations;2.18.3
-springboot3_com.fasterxml.jackson.core:jackson-core;2.18.3
-springboot3_com.fasterxml.jackson.core:jackson-databind;2.18.3
-springboot3_com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.18.3
-springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jdk8;2.18.3
-springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.18.3
-springboot3_com.fasterxml.jackson.module:jackson-module-afterburner;2.18.3
-springboot3_com.fasterxml.jackson.module:jackson-module-parameter-names;2.18.3
+springboot3_com.fasterxml.jackson.core:jackson-annotations;2.18.4
+springboot3_com.fasterxml.jackson.core:jackson-core;2.18.4.1
+springboot3_com.fasterxml.jackson.core:jackson-databind;2.18.4
+springboot3_com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.18.4
+springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jdk8;2.18.4
+springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.18.4
+springboot3_com.fasterxml.jackson.module:jackson-module-afterburner;2.18.4
+springboot3_com.fasterxml.jackson.module:jackson-module-parameter-names;2.18.4
```

Comment:
I think those changes are causing the spring CI failure, might be worth to try to revert those. I couldn't find these dependencies on maven central though?

Reply:
```suggestion
springboot3_com.fasterxml.jackson.core:jackson-annotations;2.18.3
springboot3_com.fasterxml.jackson.core:jackson-core;2.18.3
springboot3_com.fasterxml.jackson.core:jackson-databind;2.18.3
springboot3_com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.18.3
springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jdk8;2.18.3
springboot3_com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.18.3
springboot3_com.fasterxml.jackson.module:jackson-module-afterburner;2.18.3
springboot3_com.fasterxml.jackson.module:jackson-module-parameter-names;2.18.3
```